source/features/view-markdown-source.tsx

```diff
@@ -10,7 +10,9 @@ import fetchDom from '../libs/fetch-dom';
 import * as icons from '../libs/icons';
 import {blurAccessibly} from './comment-fields-keyboard-shortcuts';
 
-const linkedDom = Symbol('Attached RGH dom');
+const linkedSourceDom = Symbol('Attached RGH source dom');
```

Comment:
I believe I implemented this functionality properly with a `WeakMap`. Mind just giving it a double check?